commit 706bff0fe050bdbeba93223da7e350c1c1ae7cd5
Author: Kelly McLaughlin <kelly.mclaughlin@joyent.com>
Date:   2019-07-10T12:27:39-06:00 (3 months ago)
    
    MANTA-4399 Tweak to electric-boray configuration so DNS registration works

diff --git a/main.js b/main.js
index cfdc07a..6e3d8cd 100644
--- a/main.js
+++ b/main.js
@@ -30,7 +30,7 @@ var DEFAULTS = {
     statusPort: 4020,
     bindip: '0.0.0.0'
 };
-var NAME = 'electric-moray';
+var NAME = 'electric-boray';
 var LOG_SERIALIZERS = {
     err: bunyan.stdSerializers.err
 };
diff --git a/sapi_manifests/electric-boray/template b/sapi_manifests/electric-boray/template
index 216264c..33c0993 100644
--- a/sapi_manifests/electric-boray/template
+++ b/sapi_manifests/electric-boray/template
@@ -6,6 +6,10 @@
             "type": "udp"
         }
     },
+    "fast": {
+        "client_crc_mode": 3,
+        "server_crc_mode": 2
+    },
     "numWorkers": 0,
     "audit": false,
     "cache": {
diff --git a/sapi_manifests/registrar/template.in b/sapi_manifests/registrar/template.in
index 280b55e..a38588f 100644
--- a/sapi_manifests/registrar/template.in
+++ b/sapi_manifests/registrar/template.in
@@ -1,7 +1,7 @@
 {
   "registration": {
     "domain": "{{SERVICE_NAME}}",
-    "type": "boray_host",
+    "type": "moray_host",
     "service": {
       "type": "service",
       "service": {
